packages/payload-cloud/src/plugin.ts

```diff
@@ -93,5 +95,69 @@ export const payloadCloudPlugin =
       })
     }
 
+    // We set up cron jobs on init.
+    // We also make sure to only run on one instance using a instance identifier stored in a global.
+
+    // If you modify this defaults, make sure to update the TsDoc in the types file.
+    const DEFAULT_CRON = '* * * * *'
+    const DEFAULT_LIMIT = 10
+    const DEFAULT_CRON_JOB = {
+      cron: DEFAULT_CRON,
+      limit: DEFAULT_LIMIT,
+      queue: 'default (every minute)',
```

Comment:
Any specific reason we want to put every minute on the end here? What if they change the cron to no longer be every minute and this stays?

Reply:
This only ends up in the DB if the user does not define any cronJob. See line 142